dashboards/jupyterhub.jsonnet

```diff
@@ -35,7 +35,9 @@ local currentRunningUsers = graphPanel.new(
   prometheus.target(
     |||
       sum(
-        kube_pod_status_phase{phase="Running"}
+        group(
+          kube_pod_status_phase{phase="Running"}
+        ) by (pod,namespace)
```

Comment:
```suggestion
        ) by (pod, namespace)
```

Reply:
Hmm, given we're already doing a sum here, which will drop all labels other than `phase`, is `group` necessary?